app/code/community/Bolt/Boltpay/sql/bolt_boltpay_setup/upgrade-1.4.1-1.4.1.1.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
can you confirm if this script works when we upgrade 1.4.1 to 2.0.0 as well?

Reply:
Yes